sklearn/cluster/mean_shift_.py

```diff
@@ -65,10 +71,31 @@ def estimate_bandwidth(X, quantile=0.3, n_samples=None, random_state=0):
 
     return bandwidth / X.shape[0]
 
+#separate function for each seed's iterative loop
+def _iter_loop(my_mean,X,nbrs,max_iter):
```

Comment:
please give this function a more telling name, like ``_mean_shift_single_seed`` or ``_mean_shift_descent`` or something.